lib/miq_expression.rb

```diff
@@ -977,23 +983,24 @@ def self.get_column_details(column_names, class_path, assoc_path, opts)
       excludes += ["^.*derived_host_count_off$", "^.*derived_host_count_on$", "^.*derived_vm_count_off$", "^.*derived_vm_count_on$", "^.*derived_storage.*$"]
     end
 
-    column_names.collect do|c|
+    column_names.collect do |c|
       # check for direct match first
       next if excludes.include?(c) && !EXCLUDE_EXCEPTIONS.include?(c)
 
       # check for regexp match if no direct match
       col = c
-      excludes.each do|excl|
-        if c.match(excl)
-          col = nil
-          break
+      unless EXCLUDE_EXCEPTIONS.include?(c)
```

Comment:
Actually I think this is equivalent to before, I was seeing something else ;) I'm finding this way too confusing to think about, perhaps I can look into this in a follow up?